api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -225,4 +225,12 @@ private void logAdminAction(Long targetUserId, Long targetWorkspaceId, String ta
     adminActionHistory.setTimestamp();
     adminActionHistoryDao.save(adminActionHistory);
   }
+
+  public boolean getContactEmailTaken(String contactEmail) {
+    User user = userDao.findUserByContactEmail(contactEmail);
+    if(user != null) {
+      return true;
+    }
+    return false;
```

Comment:
Can probably be simplified to `return userDao.findUserByContactEmail(contactEmail) != null;`